core/cmd/shell.go

```diff
@@ -100,6 +101,35 @@ func initGlobals(cfgProm config.Prometheus, cfgTracing config.Tracing, cfgTeleme
 				attributes = append(attributes, attribute.String(k, v))
 			}
 
+			// note: due to the OTEL specification, all histogram buckets
+			// must be defined when the beholder client is created
+			globalMetricViews := []metric.View{
```

Comment:
Should it be configurable in chainlink core config? E.g 
```
[Telemetry.Metric.Views]
platform_engine_workflow_earlyexit_time_seconds = [0, 1, 10, 100]
```

Reply:
Should these views live next to the metrics which they correspond to?
Maybe somewhere in this file [here](https://github.com/smartcontractkit/chainlink/blob/030fd7c5309b43148a637fd762bf346467275bc6/core/services/workflows/monitoring.go#L95-L131) ?

Reply:
That was my initial approach; i wanted the consumer to have control over the buckets. Why this does not work I've documented here: https://smartcontract-it.atlassian.net/browse/KS-592

Reply:
I was thinking of something like this https://github.com/smartcontractkit/chainlink/pull/15515/commits/c082a97846d5c8e1f8f0c951a53fa7b92a8f51b0